examples/models/transform_jitter.py

```diff
@@ -1,3 +1,12 @@
+''' This example demonstrates how to add jitter to a scatter plot using Bokeh and NumPy.
```

Comment:
Numpy is not really a focus of this example, it's just used to make the raw coords, not the jitter. Suggest this change:
```suggestion
''' This example demonstrates how to us a jitter transform on coordinate data.